packages/api/cms-api/src/auth/guards/comet.guard.ts

```diff
@@ -8,7 +8,8 @@ import { isObservable, lastValueFrom } from "rxjs";
 import { CurrentUserInterface } from "../current-user/current-user";
 import { allowForRoleMetadataKey } from "../decorators/allow-for-role.decorator";
 
-function CometAuthGuard(type?: string | string[]): Type<IAuthGuard> {
+export function createCometAuthGuard(type?: string | string[]): Type<IAuthGuard> {
```

Comment:
the pervisouly used public export was GlobalAuthGuard (CometAuthGuard was used internally tough)

wouldn't it make sense to use that naming here too?

Reply:
This guard isn't necessarily the global guard, the choice is up to the application. It's just a guard which implements the Comet-way of authentication features. Furthermore the term "global" isn't used anymore, now it's called "APP_GUARD". So in my opinion the decision is between `AppGuard` and `CometGuard`, my voice goes to the latter one.